### ISSUE TITLE
pangolearn does not escape commas in sequence IDs

### ISSUE THREAD

**Issue by yp** [cov-lineages/pangolin#316]
Hi,
I am using pangolin for assignment. I have the following issue when the sequences that I use have a comma in the header.
```
RuleException:
ValueError in line 144 of /opt/conda/envs/pangolin/lib/python3.8/site-packages/pangolin/scripts/pangolearn.smk:
could not convert string to float: 'P.1'
  File "/opt/conda/envs/pangolin/lib/python3.8/site-packages/pangolin/scripts/pangolearn.smk", line 144, in __rule_add_failed_seqs
  File "/opt/conda/envs/pangolin/lib/python3.8/concurrent/futures/thread.py", line 57, in run
Exiting because a job execution failed. Look above for error message
```
My opinion is that line https://github.com/cov-lineages/pangolin/blob/b8b69be65aff9a543adbf114ab9bee9abe3d6831/pangolin/pangolearn/pangolearn.py#L197 should escape `seqId`.

Moving to module `csv` would be better in the long term.

Thanks for your work and assistance.

**Comment by aineniamh** [comment]
That's interesting, I had thought we explicitly handle commas in the sequence ids when we qc the sequences: https://github.com/cov-lineages/pangolin/blob/b8b69be65aff9a543adbf114ab9bee9abe3d6831/pangolin/command.py#L285
But it looks like it's not getting pushed through. It must just be most people don't put commas in a sequence ID. 

I've added a quick fix to handle commas in the description and will release a new version, but we can look into adopting the csv module in the pangolearn.py script as we use it everywhere else anyway.

**Comment by yp** [comment]
Thanks for the fix: this solves both #315 and #316 in my case.
Curiously, I always had commas in the sequence headers, but pangolin previous versions correctly handled the case.
